streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -396,28 +403,37 @@ public void shouldCloseProcessor() {
 
     @Test
     public void shouldThrowForUnknownTopic() {
-        final String unknownTopic = "unknownTopic";
-        final byte[] nullValue = null;
-
         testDriver = new TopologyTestDriver(new Topology(), config);
```

Comment:
This is not related to change in this PR, but after second read, should we setup non-empty topology here.
to prevent IllegalStateException to be thrown due to empty topology after some future change.
```suggestion
         testDriver = new TopologyTestDriver(setupSourceSinkTopology(), config);
```

Reply:
If we really don't allow an empty topology, and `TopologyTestDriver` constructor throw, the test would fail. Hence, such a change would break the test and we could update it then.